client/js/components/input.js

```diff
@@ -6,30 +6,28 @@ import Spacer from '../components/spacer'
 const Input = ({ classes, input, label, type, placeholder }) => {
 	return (
 		<div className={classes.child}>
-			<label className={classes.label}>{label}</label>
-			<input className={classes.textInput} type={type} placeholder={placeholder} {...input} />
+			<input
+				className={classes.textInput}
+				type={type}
+				placeholder={placeholder}
+				{...input}
+			/>
 			<Spacer height="10px" />
 		</div>
 	)
 }
 
 const styles = {
 	child: {
-		'justify-content': 'center',
-		width: '80%',
-	},
-	label: {
-		padding: '20px 10px',
-		display: 'block',
-		float: 'left',
-		width: '20%',
-		'text-align': 'right',
+		display: 'flex',
+		'align-items': 'flex-start',
+		width: '100%',
```

Comment:
It doesn't look like this `width` is actually doing anything. Flexbox and Grid seem to be taking care of the width.